README.md

```diff
@@ -88,7 +88,7 @@ Participate in the contest by registering on the EvalAI challenge page and creat
 
 - Step 4: 
 
-  Download the challenge data by completing the user agreement (https://forms.gle/ecyoPtEcCBMrQ3qF9).
+  Download the challenge data by completing the user agreement (https://forms.gle/ecyoPtEcCBMrQ3qF9). Place `ig_dataset` and `igibson.key` under `BehaviorChallenge2021` and set its permission `chmod -R 777 ig_dataset`.
```

Comment:
Is the default permission not already user readable?

Reply:
Currently mesh renderer will create new directory called `tmp` in `ig_dataset`, so write permission is needed.

Reply:
I thought it should be writeable by user by default, that's strange

Reply:
When you are in docker, you are no longer under your username right?

Reply:
Typically you would be under the system docker user which should run as root

Reply:
Yes, but for some reason, even as root, I cannot mkdir in `ig_dataset`, not sure why.

Reply:
Maybe you can give it a try on your machine? mount ig_dataset in one of your docker and try to create `tmp` in it? Maybe something is weird for my machine.

Reply:
removed